scopus/scopus_api.py

```diff
@@ -599,7 +599,7 @@ def __init__(self, ISSN, refresh=False):
         SNIP = get_encoded_text(self.results,
                                 'entry/SNIPList/SNIP')
         SNIP_year = self.results.find('entry/SNIPList/SNIP', ns)
-        if SNIP_year:
+        if SNIP_year is not None:
```

Comment:
I got an error-message, staiting I should write is not None.
For isbn you already have is not None